servers/physics_2d/broad_phase_2d_bvh.cpp

```diff
@@ -110,7 +110,4 @@ BroadPhase2DSW *BroadPhase2DBVH::_create() {
 BroadPhase2DBVH::BroadPhase2DBVH() {
 	bvh.set_pair_callback(_pair_callback, this);
 	bvh.set_unpair_callback(_unpair_callback, this);
-	pair_callback = nullptr;
-	pair_userdata = nullptr;
-	unpair_userdata = nullptr;
 }
```

Comment:
This might be worth reviewing @pouleyKetchoupp @lawnjelly as the previous code seems a bit weird. We set callbacks in the `bvh` manager before setting only the `pair_callback` to `nullptr` and leaving `unpair_callback` uninitialized.

Reply:
Actually this is fine but the code is hard to read :)

It's just that there's `_pair_callback` (with underscore) passed to the bvh and `pair_callback` (without) which is initialized to nullptr.

`unpair_callback` was uninitialized though, so it's all good now.